src/Insightly/Item/Tag.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Insightly\Item;
+
+class Tag extends Entity
```

Comment:
The serialize method is not implemented for Tag. Name will not be returned when serializing